service-admin/docker/app/health-check.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
```

Comment:
Can we have this script in one place and copy it into the two images, as it's identical between them?